### PR TITLE
[Task][29424] Attach file to tag

### PR DESCRIPTION
**Pull request by ntbai**
https://edu-redmine.sun-asterisk.vn/issues/29424

### CODE REVIEW

**Comment by ntbai** [comment]
ready

File: db/migrate/20200915062855_create_attachments.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class CreateAttachments < ActiveRecord::Migration[6.0]
   def change
     create_table :attachments do |t|
       t.string :name
-      t.text :link
+      t.text :content
```
Comment (ngocvt-0484) [inline db/migrate/20200915062855_create_attachments.rb]: tạo file migrate ms e nhé
k baoh dc sửa file migrate cũ

Comment (ntbai) [reply db/migrate/20200915062855_create_attachments.rb]: Vâng ạ

**Comment by ntbai** [comment]
ready

**Pull request merged**